### PR TITLE
[ODS-6195] Pkg EdFi.ProjectTemplates.Installer Added

### PR DESCRIPTION
**Pull request by semalaiappan**
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/pull/79

Needs Approval for this PR also

### CODE REVIEW

**Review by axelmarquezh — CHANGES_REQUESTED** [review]
Could you also update the next places that mention net6?
- [CodeGeneration/README.md](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Utilities/CodeGeneration/README.md)
- [Ed-Fi-Databases/README.md](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/blob/main/README.md)
- [Ed-Fi-Databases/.vscode/launch.json](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/blob/main/.vscode/launch.json)

**Comment by semalaiappan** [comment]
> Could you also update the next places that mention net6?
> 
> * [CodeGeneration/README.md](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Utilities/CodeGeneration/README.md)
> * [OdsStartupBase.cs](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs#L447)
> * [Ed-Fi-Databases/README.md](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/blob/main/README.md)
> * [Ed-Fi-Databases/.vscode/launch.json](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Databases/blob/main/.vscode/launch.json)

I fixed this in all places

**Review by axelmarquezh — APPROVED** [review]

File: build.githubactions.ps1 [hunk 1]
```diff
@@ -232,30 +232,35 @@ function Get-IsWindows {
...
-         Install-Module -Force -Scope CurrentUser -Name 7Zip4Powershell
+    if (Get-IsWindows)  {
+        $moduleNames = Get-Module -ListAvailable | Select-Object -ExpandProperty Name
+        if ($moduleNames -contains '7Zip4Powershell') {
+            Write-Host "'7Zip4Powershell' exists in available modules."
+        } else {
+            Install-Module -Force -Scope CurrentUser -Name 7Zip4Powershell
+            Write-Host "Installed 7Zip4Powershell."
+        }
+    }
```
Comment (axelmarquezh) [inline build.githubactions.ps1:243]: Can we revert the changes made to this condition? We have this condition in many of our PowerShell scripts, so it should work as it was.
Did you get an error?

**Review by axelmarquezh — CHANGES_REQUESTED** [review]

File: build.githubactions.ps1 (hunk 1 above)
Comment (semalaiappan) [reply build.githubactions.ps1:243]: Yes I am geting error

Comment (semalaiappan) [reply build.githubactions.ps1]: I fixed it . Thanks

**Review by axelmarquezh — APPROVED** [review]

**Review by vimayya — APPROVED** [review]
